gibbon-rts/rts-c/gibbon_rts.h

```diff
@@ -747,19 +747,19 @@ INLINE_HEADER void gib_grow_region(char **writeloc_addr, char **footer_addr)
     bool old_chunk_in_nursery;
     GibOldgenChunkFooter *old_footer = NULL;
 
-    if (gib_addr_in_nursery(footer_ptr)) {
-        old_chunk_in_nursery = true;
-        GibNurseryChunkFooter oldsize = *(GibNurseryChunkFooter *) footer_ptr;
-        newsize = oldsize * 2;
-    } else {
+    // if (gib_addr_in_nursery(footer_ptr)) {
+    //     old_chunk_in_nursery = true;
+    //     GibNurseryChunkFooter oldsize = *(GibNurseryChunkFooter *) footer_ptr;
+    //     newsize = oldsize * 2;
+    // } else {
```

Comment:
This looks like the second major thing, but it's a pity I don't understand the motivation of this change.

Reply:
The nursery is initialized in the main function when it calls "gib_init". Without initialization, gib_addr_in_nursery will segfault. We need to find a way to call "gib_init" function in a different way.